src/main/java/org/osc/manager/ism/entities/DeviceEntity.java

```diff
@@ -40,14 +43,12 @@
     @Column(name = "name", unique = true, nullable = false)
     private String name;
 
-    @OneToMany(mappedBy = "device", fetch = FetchType.LAZY)
+    @OneToMany(mappedBy = "device", fetch = FetchType.EAGER)
     private List<DeviceMemberEntity> deviceMembers = new ArrayList<DeviceMemberEntity>();
```

Comment:
add line comment on why we made this a list

Reply:
@arvindn05 , will add the comment why Set is used for SG and SGI which causes sql bag exception if we use it as a list.